test-data/unit/check-attr.test

```diff
@@ -577,7 +577,10 @@ class C:
 
 [builtins fixtures/list.pyi]
 
+-- This is tricky with new analyzer:
+-- We need to know the analyzed type of a function while still processing top-level.
```

Comment:
Probably worth creating an issue to track this.

Reply:
Opened #6438